src/main/java/alns/Evaluator.java

```diff
@@ -1,24 +1,29 @@
 package alns;
 
 import data.Problem;
+import objects.Installation;
 import objects.Order;
+import subproblem.ArcGeneration;
+import utils.DistanceCalculator;
+import utils.Helpers;
 
+import java.util.LinkedList;
 import java.util.List;
 
 public class Evaluator {
 
     public static boolean evaluateLoad(Solution solution) {
         List<List<Order>> orderSequences = solution.getOrderSequences();
-        for(int vesselNumber = 0; vesselNumber < orderSequences.size(); vesselNumber++) {
+        for (int vesselNumber = 0; vesselNumber < orderSequences.size(); vesselNumber++) {
             double currentLoad = findTotalStartLoad(orderSequences.get(vesselNumber));
-            if(currentLoad > Problem.vessels.get(vesselNumber).getCapacity()) return false;
-            for(Order order : orderSequences.get(vesselNumber)) {
-                if(order.isDelivery()) {
+            if (currentLoad > Problem.vessels.get(vesselNumber).getCapacity()) return false;
```

Comment:
I stedet for å gette vessel hver gang du trenger den under her, lag en variabel `Vessel vessel = Problem.vessels.get(vesselNumber)` på linje 18